echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/bearychat/BearychatUserInfo.groovy

```diff
@@ -0,0 +1,28 @@
+package com.netflix.spinnaker.echo.bearychat
+
+class BearychatUserInfo {
+  String  inactive
```

Comment:
Please don't space align the field names

Reply:
OK. I will change this coding style. And i am just graduate from university, and my major is EE. I dont have much coding experience. Would you please tell me the reason that don't space align the field names?  Is it to make code looks good? or it is just the coding style you are using?

Reply:
No worries about experience, we're all here to learn

One reason is that it's just the coding style we use, and it's best to be consistent.

Another reason is that if you have a lot of space-aligned fields, and you add a new one with a longer type qualifier, you have to realign the whole class, which is a lot of extra work and code to review later.

Reply:
Oh, I see. Thank you!